src/Watch.php

```diff
@@ -30,7 +30,7 @@ public function __construct(Source $src, array $tasks)
                             'yellow'
                         );
                         $distFile->setLastChangeTime($timeChange);
-                        Phulp::start($tasks);
+                        $callback();
```

Comment:
The `Phulp` object **is** referenced, butt _hidden_ inside the closure. See [here](https://github.com/reisraff/phulp/pull/14/files#diff-389a38f7e57a5cccdb6a170420f68bc0R77) for how I set up the closure/callback.